src/music_service.py

```diff
@@ -0,0 +1,269 @@
+
+import dbus
+import platform
+from dbus.mainloop.glib import DBusGMainLoop
+
+class Singleton(type):
+    _instances = {}
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
+        return cls._instances[cls]
+
+class MusicService(metaclass=Singleton):
+    def __init__(self):
+        self.UUID_STATUS = '00000002-78fc-48fe-8e23-433b3a1942d0'
```

Comment:
Very small (and optional) nit, but since these are constants, we could define them at the class level (right below line 13) instead of inside `__init__ `

Reply:
done